packages/plugin-core/src/commands/CreateNoteWithTraitCommand.ts

```diff
@@ -110,6 +110,11 @@ export class CreateNoteWithTraitCommand extends BaseCommand<
       title = this.trait.OnCreate.setTitle(context);
     }
 
+    let body;
+    if (this.trait.OnCreate?.setBody && this.checkWorkspaceTrustAndWarn()) {
```

Comment:
can we be more specific on the error? `checkWorkspaceTrustAndWarn` should take a helper text so we can tell the user which method failed (eg. `onCreate.setBody` vs `onCreate.setTitle`)

Reply:
I think the error msg is correct - if workspace isn't trusted, all custom note-trait behavior should be blocked, irrespective of whether its setBody logic, setTitle logic, or something else. 

There is a problem here though where it'll pop the toast multiple times; I've fixed that now.